osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -558,7 +558,9 @@ private Channel joinChannel(Channel channel, bool fetchInitialMessages = false)
         /// Leave the specified channel. Can be called from any thread.
         /// </summary>
         /// <param name="channel">The channel to leave.</param>
-        public void LeaveChannel(Channel channel) => Schedule(() =>
+        public void LeaveChannel(Channel channel) => Schedule(() => leaveChannel(channel, true));
+
+        private void leaveChannel(Channel channel, bool sendLeaveRequest)
```

Comment:
This new parameter doesn't look to add any new effect, am I missing something?

Reply:
I mean it looks like it should be next to the `channel.Joined.Value` check at the end of this method, but you're right that it just... isn't.

Reply:
Sorry, having issues with rider not saving files across the board (do not use EAP).